C/tests/ParserTest_0000_0999.cpp

```diff
@@ -2015,102 +2015,464 @@ void ParserTest::case0279()
+          Expectation().AST({ TranslationUnit,
+                              FunctionDefinition,
+                              BuiltinTypeSpecifier,
+                              BuiltinTypeSpecifier,
+                              FunctionDeclarator,
+                              IdentifierDeclarator,
+                              ParameterSuffix,
+                              ParameterDeclaration,
+                              TypedefName,
+                              AbstractDeclarator,
+                              ExtKR_ParameterDeclaration,
+                              BuiltinTypeSpecifier,
+                              BuiltinTypeSpecifier,
+                              PointerDeclarator,
+                              IdentifierDeclarator,
+                              CompoundStatement,
+                              ReturnStatement,
+                              PointerIndirectionExpression,
+                              IdentifierName }));
 
 }
 
 void ParserTest::case0292()
 {
-
+    parse("float x ( y , z ) int y ( float [ 2 ] ) ; float * z  ; { return y ( z ) ; }",
+          Expectation().AST({ TranslationUnit,
+                              FunctionDefinition,
+                              BuiltinTypeSpecifier,
+                              FunctionDeclarator,
+                              IdentifierDeclarator,
+                              ParameterSuffix,
+                              ParameterDeclaration,
+                              TypedefName,
+                              AbstractDeclarator,
+                              ParameterDeclaration,
+                              TypedefName,
+                              AbstractDeclarator,
+                              ExtKR_ParameterDeclaration,
+                              BuiltinTypeSpecifier,
+                              FunctionDeclarator,
+                              IdentifierDeclarator,
+                              ParameterSuffix,
+                              ParameterDeclaration,
+                              BuiltinTypeSpecifier,
+                              ArrayDeclarator,
+                              SubscriptSuffix,
+                              IntegerConstantExpression,
+                              ExtKR_ParameterDeclaration,
+                              BuiltinTypeSpecifier,
+                              PointerDeclarator,
+                              IdentifierDeclarator,
+                              CompoundStatement,
+                              ReturnStatement,
+                              CallExpression,
+                              IdentifierName,
+                              IdentifierName }));
 }
 
 void ParserTest::case0293()
 {
-
+    parse("int x ( y , z ) int y , int z { return y + z ; }",
+          Expectation().diagnostic(
+              Expectation::ErrorOrWarn::Error,
+              Parser::DiagnosticsReporter::ID_of_ExpectedFOLLOWofDeclarator
+          ).diagnostic(
```

Comment:
Coding style: `)` should be at the end of line above and `.diagnostic` indented as the other(s)  `.diagnostic`.